crates/rolldown/tests/fixtures/semantic/export_star_from_external_as_shared_entries_cjs/artifacts.snap

```diff
@@ -43,6 +43,7 @@ require("node:fs");
 //#region main.js
 var main_ns = {};
 __reExport(main_ns, require("node:fs"));
+__reExport(main_ns, require("node:fs"));
```

Comment:
Not related to this pr, this duplicate `__reExport` is not generate in module level.